public/app/features/live/dashboard/DashboardChangedModal.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { PureComponent } from 'react';
+import { Modal, VerticalGroup } from '@grafana/ui';
+import { css } from 'emotion';
+import { dashboardWatcher } from './dashboardWatcher';
+import { config } from '@grafana/runtime';
+import { DashboardEvent, DashboardEventAction } from './types';
+
+interface Props {
+  event?: DashboardEvent;
+}
+
+interface State {
+  dismiss?: boolean;
+}
+
+interface ActionInfo {
+  label: string;
+  description: string;
+  action: () => void;
+}
+
+export class DashboardChangedModal extends PureComponent<Props, State> {
+  state: State = {};
+
+  discardAndReload: ActionInfo = {
+    label: 'Discard local changes',
+    description: 'Load the latest saved version for this dashboard',
+    action: () => {
+      dashboardWatcher.reloadPage();
+      this.onDismiss();
+    },
+  };
+
+  continueEditing: ActionInfo = {
+    label: 'Continue editing',
+    description:
+      'Keep your local changes and continue editing.  Note: when you save, this will overwrite the most recent chages',
+    action: () => {
+      this.onDismiss();
+    },
+  };
+
+  acceptDelte: ActionInfo = {
+    label: 'Discard Local changes',
+    description: 'view grafana homepage',
+    action: () => {
+      // Navigate to the root URL
+      document.location.href = config.appUrl;
+    },
+  };
+
+  onDismiss = () => {
+    this.setState({ dismiss: true });
+  };
+
+  render() {
+    const { event } = this.props;
+    const { dismiss } = this.state;
+    const radioClass = css`
+      cursor: pointer;
+      width: 100%;
+      padding: 20px;
+      background: ${config.theme.colors.dropdownBg};
+
+      &:hover {
+        background: ${config.theme.colors.formCheckboxBgCheckedHover};
+      }
+    `;
```

Comment:
this feels like a new design system component. Not sure what to do for this PR if there is no current component that fits. 

For a new component & we should use useTheme & useStyles